### PR TITLE
Package Kubernetes v1.11.7 client binary 

### PR DESCRIPTION
**Pull request by zacblazic**
As per the title.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 FROM alpine:3.8
 
-LABEL maintainer "Zappistore Devops <devops@zappistore.com>"
+LABEL maintainer "Zappi Devops <devops@zappistore.com>"
```
Comment (hadrianvalentine) [inline Dockerfile:3]: That brand alignment though

File: circle.yml [hunk 2]
```diff
@@ -1,14 +0,0 @@
----
```
Comment (hadrianvalentine) [inline circle.yml:1]: Bonus points for cleanup

**Review by hadrianvalentine — APPROVED** [review]
LGTM 👍

**Comment by itskingori** [comment]
LGTM.

**Pull request merged**